vendor/k8s.io/kube-openapi/pkg/generators/openapi.go

```diff
@@ -383,8 +383,9 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 		if hasOpenAPIDefinitionMethods(t) {
 			g.Do("return $.OpenAPIDefinition|raw${\n"+
 				"Schema: spec.Schema{\n"+
-				"SchemaProps: spec.SchemaProps{\n"+
-				"Type:$.type|raw${}.OpenAPISchemaType(),\n"+
+				"SchemaProps: spec.SchemaProps{\n", args)
+			g.generateDescription(t.CommentLines)
+			g.Do("Type:$.type|raw${}.OpenAPISchemaType(),\n"+
```

Comment:
Is this really not gofmt'd in kube-openapi? @mbohlool or @roycaihw can one of you check if kube-openapi is missing some presubmit test?

Reply:
That is formatted… was there something different you were expecting? Gofmt is run on the generated code after generation as well